ovos_plugin_manager/templates/audio.py

```diff
@@ -90,6 +89,7 @@ def _uri2meta(uri):
             from ovos_ocp_files_plugin.plugin import OCPFilesMetadataExtractor
             return OCPFilesMetadataExtractor.extract_metadata(uri)
         except:
+            from ovos_utils.ocp import PlaybackType, TrackState
```

Comment:
this one existed before utils 0.1.0, so it should probably redeclare the enums on import fail, we don want to start throwing an exception here